Source/WebCore/css/CSSFontFaceSet.cpp

```diff
@@ -117,19 +117,21 @@ void CSSFontFaceSet::ensureLocalFontFacesForFamilyRegistered(const AtomString& f
     if (m_locallyInstalledFacesLookupTable.contains(familyName))
         return;
 
-    if (!m_owningFontSelector->scriptExecutionContext())
+    Ref protectedOwningFontSelector { *m_owningFontSelector };
```

Comment:
We typically don't call these protectedFoo anymore. Also, we usually use = instead.